.travis.yml

```diff
@@ -8,8 +8,10 @@ php:
   - 5.4
 
 install:
-  - composer install --dev --prefer-source
+  - composer install --prefer-source
 
 script:
   - bin/phpunit
   - bin/psecio-parse scan src tests
+
+dist: trusty
```

Comment:
1. is this even needed?
2. I recommend moving this on top after `language: php` to increase visibility if it's actually needed.